types/tx.go

```diff
@@ -236,6 +238,16 @@ func (txs Txs) FilterTransactionsByType(types []TransactionType) Txs {
 	return result
 }
 
+func (txs Txs) FilterTransactionsByTokenTransfersInContract() Txs {
```

Comment:
I don't think this method is necessary here

Reply:
I actually thought the same, let's move it to blockatlas then

Reply:
feel free to delete it, I don't think I need it for that change